src/hmrc-content-guide/customs-information-papers/index.njk

```diff
@@ -52,17 +52,31 @@ Description (CIP #)
 </p>
 
 <p class="govuk-body">
-Example: Cessation of the printed tariff (CIP 12)
+Example: Goods arriving in Great Britain without an import declaration (CIP 1)
 </p>
 
 <h3 class="govuk-heading-m">Summary</h3>
 
 <p class="govuk-body">
-Example of a summary: Find out about the end of production of the printed paper Integrated Customs Tariff in December 2019.
+Example of a summary: HMRC’s policy for importers who have unintentionally received goods from 1 January 2021 to 31 December 2021 in Great Britain, before they’ve had the opportunity to declare them.
 </p>
 
 <h3 class="govuk-heading-m">Collection page</h3>
 
 <p class="govuk-body">
 Add the document to the <a href="https://www.gov.uk/government/collections/customs-information-papers--2" class="govuk-link">Customs information papers</a> collection page. Do not create a public change note.
+</p>
+
+<h2 class="govuk-heading-l">HTML attachment</h2>
```

Comment:
Please make 'HTML attachment' under 'How it works' in the title structure. See http://localhost:3000/hmrc-content-guide/anti-dumping-measures/ as an example.